### ISSUE TITLE
Failed to load plugin 'jsx-a11y': Cannot read property 'elementsFromPoint' of undefined

### ISSUE THREAD

**Issue by ostowe** [jsx-eslint/eslint-plugin-jsx-a11y#722]
## Summary
I began getting this error quite recently and it's causing eslint to fail to execute. Here's the full text of the error `Failed to load plugin 'jsx-a11y' declared in '--config » eslint-config-airbnb » /Users/owenstowe/.../node_modules/eslint-config-airbnb/rules/react-a11y.js': Cannot read property 'elementsFromPoint' of undefined`

I'm posting this issue here because my hunch is this is an error coming from `axe-core`. Axe core is the only package in `node_modules` where I could find any reference to `elementsFromPoint`.

Is there a workaround for this issue or have you encountered it before?

## Versions/Environment
As you can see in the the above error, I'm using `eslint-plugin-jsx-a11y` via `eslint-config-airbnb`. Below are the versions of all the eslint packages I have installed:
```
    "eslint": "^7.3.1",
    "eslint-config-airbnb": "^18.2.0",
    "eslint-import-resolver-babel-module": "^5.1.2",
    "eslint-loader": "^4.0.2",
    "eslint-plugin-import": "^2.22.0",
    "eslint-plugin-jsx-a11y": "^6.3.1",
    "eslint-plugin-react": "^7.20.3",
    "eslint-plugin-react-hooks": "^4.0.5",
    ...
   "webpack": "^4.43.0",
```

This is all running on Node 12 and NPM 6 via `eslint-loader` and `webpack`.

Any insight into this issue would be greatly appreciated. Thank you!

**Comment by ljharb** [comment]
Can you share the entire stack trace?

**Comment by ostowe** [comment]
@ljharb thanks for the quick response, as it turns out this is 100% on my end. After doing some further investigation there's some functionality in this app that shims the `window` object in a node environment, which probably causes `axe` to think browser-specific functionality is safe to use.
